pootle/apps/pootle_app/models/directory.py

```diff
@@ -31,7 +31,10 @@ def root(self):
 
     @cached_property
     def projects(self):
-        return self.get(pootle_path='/projects/')
+        return self.select_related(
+            "parent",
```

Comment:
@phlax, I'm confused here:
- parent is `/` root directory
- there is no translation project of `/` root directory
- `projects` is a list of projects... so `__project` has no sense